### ISSUE TITLE
[$15]The form fields on the page do not have associated labels.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3236]
**Describe the bug**

The form fields on the page do not have associated labels.

**To Reproduce**

1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 
2. Type in each form field to check if there is a visible label at all times. 
3. Notice all the text fields have placeholder text which disappears when we start typing.

**Expected behavior**

There should be a visible label at all times which is associated with the respective form field.

**Actual behavior**
The placeholder text has been given for all the form fields which disappears when user starts typing.

**Additional context**

Type: Form Fields

Page: https://accounts.topcoder.com/member/registration

Status: Not Fixed

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099925 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099925 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099925 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/304

**Comment by drasticdpk** [comment]
@veshu @sushilshinde @lakshmiathreya ,
Can we create another issue over here (Which does not fall on current issue's scope) ?
Suggestion : The space between the control is not consistent. And the label should **USER NAME** .
Here is the reference:
![image](https://user-images.githubusercontent.com/25297859/64431804-dcf4f600-d0da-11e9-901b-f5c16a7e09bd.png)

**Comment by r0hit-gupta** [comment]
@drasticdpk it kind of follows the existing spacing without labels. Have a look below.

![image](https://user-images.githubusercontent.com/16841686/64440291-5a276780-d0e9-11e9-97c7-3b02e5277597.png)

**Comment by drasticdpk** [comment]
Hi @r0hit-gupta , thats why I updated as Suggestion for creating the new issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099925<br/><br/>```This is an automated message for crazyk via Topcoder X```
